test/standard_cxx_flags_test/tests.bzl

```diff
@@ -16,7 +16,9 @@ def _impl(ctx):
 
     assert_contains_once(flags.cxx_linker_executable, "-fblah3")
     assert_contains_once(flags.cxx_linker_shared, "-fblah3")
-    assert_contains_once(flags.cxx_linker_static, "-fblah3")
+    for item in flags.cxx_linker_static:
```

Comment:
I think this can be rewritten as `if "-fblah3" in flags.cxx_linker_static:`.

Reply:
Done.